bigbluebutton-html5/imports/api/voice-users/server/publishers.js

```diff
@@ -11,7 +11,7 @@ function voiceUser(credentials) {
 
   Logger.debug(`Publishing Voice User for ${meetingId} ${requesterUserId}`);
 
-  return VoiceUsers.find({ meetingId });
+  return VoiceUsers.find({ intId: requesterUserId });
```

Comment:
Are you trying to have access to the voice user from main room in the breakout?

Reply:
Yes

Reply:
Can you not just check to see if the person is in the current voice conference? When they join the breakout room's voice bridge they leave the main voice bridge.

Reply:
I don't have idea how to do this, could you give me an insight in how to do this in `HTML5`?

Reply:
Publish VoiceUsers who are both in `meetingId` and have an `intId = requesterUserId`. If you get a result then the user is in the current meeting's voiceBridge and the mute button should be visible.

Reply:
But the old connections items aren't removed, so sometimes the register returned isn't the last, I tried remove, but it bugged the echo test. I added the updadeTime to make sure that I'm getting the last item and comparing with meeting voice conf

Reply:
I'm not quite sure what you mean when you say that the item isn't the last. The collections already update when you transfer the audio to the breakout room. I just tried it on my 2.2 server and as soon as the moderator joins the breakout room audio their microphone icon in the user list is removed.